JetNews/README.md

```diff
@@ -22,8 +22,8 @@ screen uses a navigation drawer.
 
 Package [`com.example.jetnews.ui`][1]
 
-[`JetnewsApp.kt`][2] arranges the different screens in the `NavDrawerLayout`. It also implements a simple
-navigation pattern.
+[`JetnewsApp.kt`][2] arranges the different screens in the `NavDrawerLayout`. It also implements 
+the navigation library to navigate from one screen to the next.
```

Comment:
This file doesn't implement that, it's in NavGraph.kt. I'd add another line with it and rephrase as it's not "implementing the library".

Reply:
Done, can you check the rewording if this is more clear?